tokenExchange/resources/spidhub-test.dev.pn.pagopa.it_public.pem

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
```

Comment:
Per ora lo teniamo ma andrà sistemato spid-hub login aggiungendo l'url

Reply:
utilizzare l'url http://spidhub-test.dev.pn.pagopa.it:9090/.well-known/jwks.json